### PR TITLE
pubsub: consumer + test (lint)

### PR DESCRIPTION
**Pull request by mxmCherry**
Not sure if test passes - no access - agreed not to focus on making it work for now (just drafted impl).

### CODE REVIEW

File: pubsub/pubsub_test.go [hunk 1]
```diff
@@ -164,19 +204,46 @@ func teardownCB(...string) error {
...
+	if err != nil {
+		return err
+	}
+	defer topic.Stop()
+
+	for i, msg := range messages {
+		nativeMsg := &native.Message{
+			Data: msg.Data(),
+		}
+		if _, err := topic.Publish(ctx, nativeMsg).Get(ctx); err != nil {
```
Comment (mxmCherry) [inline pubsub/pubsub_test.go:244]: @dim IMO, if that works - publish message with only Data set (I've copied this from native pkg example) - we can trim the `PubMessage` struct to just `struct { Data []byte }` or even better - just accept `[]byte` directly.

Also, another observation is that `SubMessage { Data() []byte }` interfaces always goes to just wrapping `[]byte` with `type bps.RawSubMessage []byte`.
I think, we could ditch `SubMessage` interface as well - I don't think we should expose any other methods on it, like Ack/Nack - it hides nicely behind the Subscriber implementation and doing Ack/Nack by `handler.Handle(msg) error`'s result is much better (lib user won't ever forget to Ack/Nack, just return error).

File: bps.go [hunk 2]
```diff
@@ -3,6 +3,9 @@ package bps
 import "github.com/google/uuid"
 
 // GenClientID generates random client ID.
+// It is guaranteed to:
+// only start with [a-z],
```
Comment (dim) [inline bps.go:7]: you should say that it will start with `bps-`

File: bps.go [hunk 3]
```diff
@@ -3,6 +3,9 @@ package bps
...
+// only contain [0-9A-Za-z_-]
```
Comment (dim) [inline bps.go:8]: no `_` possible

File: pubsub/pubsub_test.go (hunk 1 above)
Comment (mxmCherry) [reply pubsub/pubsub_test.go:244]: @dim please, pay attention to this one as well ^^

File: bps.go (hunk 3 above)
Comment (mxmCherry) [reply bps.go:8]: I thought I'll specify it for future, but let's remove.

File: pubsub/pubsub_test.go (hunk 1 above)
Comment (mxmCherry) [reply pubsub/pubsub_test.go:244]: I'll merge as is, but this still is something to consider.

**Pull request merged**